src/games/gamemodelitem.py

```diff
@@ -17,9 +17,26 @@ def __init__(self, game, me):
         self._me = me
         self._me.relationsUpdated.connect(self._check_host_relation_changed)
 
+        # Update highlight
+        self._highlight_timer = QTimer()
+        self._highlight_timer.setSingleShot(True)
+        self._highlight_timer.setInterval(250)
+        self._highlight_timer.timeout.connect(self._game_updated)
```

Comment:
I'd connect it to a new method in case we ever want to do something more at actual game update.

Reply:
yes, splitting it is better.